test/run-docker.sh

```diff
@@ -34,18 +34,13 @@ if [[ "$(is_running boulder-mysql)" != "true" ]]; then
 	# bring up mysql mariadb container - no need to publish port
 	# 3306 with host networking
 	docker run -d \
-		--net host \
 		-e MYSQL_ALLOW_EMPTY_PASSWORD=yes \
 		--name boulder-mysql \
-		mariadb:10.0 mysqld --bind-address=127.0.0.1
+		mariadb:10.0 mysqld
 fi
 
 if [[ "$(is_running boulder-rabbitmq)" != "true" ]]; then
-	# bring up rabbitmq container - no need to publish port 5672
-	# with host networking
 	docker run -d \
-		--net host \
-		-e RABBITMQ_NODE_IP_ADDRESS=127.0.0.1 \
```

Comment:
For the same reasons as with mariadb image the patch should not expose rabbitmq over IPv6. If I understand https://www.rabbitmq.com/networking.html correctly for that it is sufficient to specify `RABBITMQ_NODE_IP_ADDRESS=0.0.0.0` here.